### ISSUE TITLE
change help exit code

### ISSUE THREAD

**Issue by reaperzn** [rackerlabs/auter#50]
Currently any auter errors that call the print_help function (All 2 of them) have a subsequent exit call which returns exit code 0:

```
paolo@paolo-Desktop:~/Scripts/auter$ grep -n "print_help " auter
151:([[ ! $1 ]] || [[ ! $1 =~ "-" ]]) && print_help && exit 0
164:    -h|--help) print_help ; exit 0; shift;;
```

Is it not standard that when calling the help function most applications use exit with non-zero? Also does it not make more sense to move the exit call to the print_help function?

**Comment by loopsysdev** [comment]
The standard in \*nix tools is that the process should exit with zero if the user specifically requested help, and non-zero if there is any problem with the given arguments.

**Comment by reaperzn** [comment]
This was resolved as part of #49 
```
175:    -h|--help) print_help ; exit 0; shift;;
189:[[ "${PREP}${APPLY}${REBOOTCALL}${POSTREBOOT}${ENABLE}${DISABLE}" == "" ]] && (print_help && exit 1)
```
